### PR TITLE
Card sort blogpost

### PR DESCRIPTION
**Pull request by shannonbux**

### CODE REVIEW

File: docs/blog/2018-06-26-card-sort-results/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+Recently, 36 Gatsby users completed an open card sort to help make [next.gatsbyjs.org](https://next.gatsbyjs.org/) easier to use. Without any outside input or help, they each sorted 90 cards into categories that made sense to them and then named those categories.
+
+### Raw & beautified data
+
+I ran this card sort through [OptimalSort](https://www.optimalworkshop.com/), a software that moderates the card sort and presents the data in usable forms. If you’re curious to see the full results, you can [view the data](https://www.optimalworkshop.com/optimalsort/x87kpp82/5x34psa3-0/shared-results/fa8b66knb66qyhwh5l8j38bd273vkkm7), including raw data as well as things like a similarity matrix and a [dendogram](https://support.optimalworkshop.com/hc/en-us/articles/201997650-Interpreting-the-OptimalSort-dendrograms).
+
+## How did participants categorize the docs?
+
+Most people sorted the docs into the following categories (I included multiple names when the some category names were equally popular):
```
Comment (calcsam) [inline docs/blog/2018-06-26-card-sort-results/index.md]: "the docs, they put them into the following categories."

parenthetical should maybe go below; eg `too surprising (I included multiple names when the some category names were equally popular)`

File: docs/blog/2018-06-26-card-sort-results/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+* Get Started / Welcome to Gatsby / Intro
+* Development environment
+* Releases & Migration
+* Core Concepts / About Gatsby
+* Advanced Guides
+* API Reference
+* Tutorials / Examples
+* Contributing
+
+These categories weren’t too surprising. However, upon further digging into the data, I discovered some unexpected results that further refine how we might categorize the docs and even write new docs.
```
Comment (calcsam) [inline docs/blog/2018-06-26-card-sort-results/index.md]: categorize the docs, write new docs, and change how we think about and talk about Gatsby.

File: docs/blog/2018-06-26-card-sort-results/index.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+## Unexpected results
```
Comment (calcsam) [inline docs/blog/2018-06-26-card-sort-results/index.md]: remove

File: docs/blog/2018-06-26-card-sort-results/index.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+Some of the results surprised me, and they have significant implications for not just how we organize the docs but also how we think about and talk about Gatsby.
```
Comment (calcsam) [inline docs/blog/2018-06-26-card-sort-results/index.md]: remove

File: docs/blog/2018-06-26-card-sort-results/index.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+No one seems to know what to do with the doc titled “Building apps with Gatsby”. In the similarity matrix data visualization, 41% of people associated it with the “Authentication tutorial” doc and the “Progressive web app” doc (second association pictured below), which was its strongest association with any other doc (and 41% isn’t *very* strong). Essentially, this means participants didn’t think it “fit” very snugly alongside any other docs. 
+
+![Building apps with Gatsby is weakly associated with other docs](building-apps-with-gatsby.png)
+
+This could mean several things:
+1. We don’t have very many docs about how Gatsby can be used to build dynamic apps.
+2. Perhaps people don’t think of Gatsby as a tool they’d use to build apps with
+3. Maybe the title of the doc is too broad or too vague
+4. Another reason I haven’t thought of yet
```
Comment (calcsam) [inline docs/blog/2018-06-26-card-sort-results/index.md]: you can leave this off, if you want a qualified, "several things, for exmple" works fine.

**Comment by calcsam** [comment]
This looks amazing! Left a couple of nitpicky edits, let's merge this & publicize!

**Comment by LeKoArts** [comment]
Hope this is the right place:
I wanna comment on the html.js as I also had problems to put it into a fitting category. The main reason is that to my knowledge the file itself shouldn’t/doesn’t need to be edited anymore in the latest version of Gatsby. Is this right?

**Comment by shannonbux** [comment]
@LeKoArts great question; I don't know the answer! @KyleAMathews?

File: docs/blog/2018-06-26-card-sort-results/index.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+
+> ***NOTE*** If you look at the data, the number of unique categorizations isn’t always a useful number, because Optimal Sort considers “getting started” and “get started” to be different categories even though a human eye can see they are essentially the same. In the case of these deployment docs, however, the 28 categories are almost all *truly unique*.
+
+### Core Concepts vs. Advanced Guides
+
+There was quite a bit of overlap in the docs people assigned to these two categories. This is probably because the two categories have a close relationship with each other; they work in harmony to help Gatsby users build at an advanced level. For example, if you read about how Gatsby works with GraphQL in “Core Concepts,” there is a high chance you’ll want to start playing with GraphQL queries and find examples of these in “Advanced Guides.” I’m not entirely sure how to reflect this close relationship in the docs; the most straightforward way is probably for each doc in “Core Concepts” to have a list of relevant links in Advanced Guides, and visa versa.
```
Comment (fk) [inline docs/blog/2018-06-26-card-sort-results/index.md]: vice versa

**Comment by shannonbux** [comment]
ok @fk incorporated your suggestions. Should we merge this?

**Comment by m-allanson** [comment]
> The main reason is that to my knowledge the file itself shouldn’t/doesn’t need to be edited anymore in the latest version of Gatsby. 

@LeKoArts I'm not sure if this is the 'official' line on `html.js`, but I think of it as an escape hatch. There's a 99% chance you won't need to use it, but it's there if you do.

**Comment by m-allanson** [comment]
Cherry-picked to `v1`

**Comment by shannonbux** [comment]
@m-allanson thanks! Does cherry picking means it's only in v1? Basically, is there anything I should do differently next time?

**Comment by m-allanson** [comment]
> Basically, is there anything I should do differently next time?

Short answer - nope!

Longer answer - We're in a temporarily odd situation where gatsbyjs.org is running from the `v1` branch, and next.gatsbyjs.org is running from the `master` branch. We want site updates to appear in both branches but there's not really a simple way to do that through GitHub. So we've been accepting changes to `master` and then manually copying them over to the `v1` branch with git (`git cherry-pick <commit>`). 

I've been commenting on each PR that I do that on (using a saved reply) so other maintainers know that it's done.

**Comment by fk** [comment]
👍

**Comment by shannonbux** [comment]
ah thanks @m-allanson!

**Pull request merged**